week10 progress - Outreachy internship experience (#208)
mixxxdj/website: content/news/2021-02-07-week-10-progess.md

title: "About my Outreachy internship with Mixxx DJ: Week 10 progress"
authors: Aanyu Deborah Oduman
tags: personal, outreachy, internship, manual
date: 2021-02-07 23:09:00

_This is the tenth weekly progress report of my Outreachy internship with Mixxx DJ._

This week:

> - I made the required edits to the YouTube tutorial about the Mixxx  communities and uploaded it to YouTube.

Reviewer: ```suggestion
- I made the necessary edits to the YouTube tutorial about the Mixxx community and uploaded it to YouTube.
```

>  Here is the link. [Get in touch with the Mixxx community](https://www.youtube.com/watch?v=0f6QrjiFNJQ)

Reviewer: ```suggestion
 [Here is the link.](https://www.youtube.com/watch?v=0f6QrjiFNJQ)
```

- I had a brief video call with community team leaders from the [Tor Project](https://www.torproject.org/) to discuss more about the [internet censorship situation](https://explorer.ooni.org/country/UG) that has been going on in Uganda.
 The purpose of the discussion was to understand more about how I am using Tor and other anti-censorship tools, and how it can be improved based on my feedback.

> - As part of the homework that I did for the cookbook chapter, I read through the first few chapters of the Mixxx manual, looking for sections that could use the step by step instruction approach that the cookbook will provide.

Reviewer: "Homework" sounds like school exercises.

Perhaps rephrase it:
```suggestion
- To get some inspiration for the cookbook chapter, I read through the first few chapters of the Mixxx manual, looking for sections that could use the step by step instruction approach that the cookbook will provide.
```

- I will be adding new sections to the same chapter with the same step by step approach – this will be my main task this week. All the (new) sections that I will be adding are subject to discussion. It is a work in progress 🙂
- Finally, I will be making another YouTube tutorial introducing Mixxx to the first time user… This tutorial will be guided by [chapter 4 of the manual](https://manual.mixxx.org/2.2/en/chapters/library.html) and will go something like;

> `Start the Mixxx software` <br>
> `Import music to the Mixxx library - import your audio files` <br>
> `Analyze your library tracks (what it means to analyze a track)` <br>
> `Configure Sound output - especially if you're not using any DJ Hardware` <br>
> `Play your first track with Mixxx` <br>

Reviewer 2: ```suggestion
  - Start the Mixxx software
  - Import music to the Mixxx library - import your audio files 
  - Analyze your library tracks (what it means to analyze a track) 
  - Configure Sound output - especially if you're not using any DJ Hardware 
  - Play your first track with Mixxx 
```
You can make nested lists in markdown by prefixing every line with two extra spaces.
https://commonmark.org/help/tutorial/10-nestedLists.html

Reviewer: Also, the `<br>` tags are unneeded and backticks are for code.

Author: Thank you! :)
